src/poliastro/czml/extract_czml.py

```diff
@@ -6,7 +6,7 @@
 from astropy.coordinates import CartesianRepresentation
 from astropy.time import Time, TimeDelta
 
-from poliastro.contrib.czml_extract_default_params import DEFAULTS
+from poliastro.czml.czml_extract_default_params import DEFAULTS
```

Comment:
I think the `ImportError` on Windows comes from the fact that `poliastro/czml` is missing an `__init__.py`, so Python understands it's a package.

Reply:
Ah you're completely right. I was really confused as to why it was failing, since everything seemed to work locally.